extension/scripts/features/customProfileHeading/ttCustomProfileHeading.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+(async () => {
+	featureManager.registerFeature(
+		"Custom Profile Heading",
+		"profile",
+		() => settings.pages.profile.customProfileHeading,
+		null,
+		customHeading,
+		removeCustomHeading,
+		{
+			storage: ["settings.pages.profile.customProfileHeading"],
+		},
+		async () => {
+			await requireElement(".profile-container .row.basic-info > *:first-child");
+		}
+	);
+
+	function customHeading() {
+		const skipToContent = document.find("h4#skip-to-content");
+		const onlOrOffIcon = document.find(".profile-container .row.basic-info > *:first-child");
+		const onlOrOffIconClone = onlOrOffIcon.cloneNode(true);
+		const backgroundPos = getComputedStyle(onlOrOffIcon).backgroundPosition;
+		onlOrOffIconClone.style.backgroundPosition = backgroundPos;
+		onlOrOffIconClone.classList.add("tt-profile-icon");
+		skipToContent.innerHTML = `${skipToContent.innerHTML.replace("'s Profile", "")} [${getUserID()}]`;
```

Comment:
This should probably be a different feature.

Reply:
That would also make this one have to change in name.

Reply:
It is actually the same thing.
https://github.com/Mephiles/torntools_extension/blob/master/torntools/scripts/content/profile/ttProfile.js#L1011

Reply:
That code doesn't alter the name?

![Settings v5](https://i.imgur.com/iLFpNKy.png)

Reply:
Done.